Prevent network access and provide the binary ourselves.
Increase test runner verbosity for better logs.

Author: Lucio Sauer <watermanpaint@posteo.net>
--- a/build/ninja_gen/src/cargo.rs
+++ b/build/ninja_gen/src/cargo.rs
@@ -147,23 +147,15 @@ pub struct CargoTest {
 
 impl BuildAction for CargoTest {
     fn command(&self) -> &str {
-        "cargo nextest run --color=always --failure-output=final --status-level=none $cargo_flags"
+        "cargo nextest run --color=always --failure-output=final --status-level=pass $cargo_flags"
     }
 
     fn files(&mut self, build: &mut impl FilesHandle) {
         build.add_inputs("", &self.inputs);
-        build.add_inputs("", inputs![":cargo-nextest"]);
         build.add_output_stamp("tests/cargo_test");
     }
 
     fn on_first_instance(&self, build: &mut Build) -> Result<()> {
-        build.add_action(
-            "cargo-nextest",
-            CargoInstall {
-                binary_name: "cargo-nextest",
-                args: "cargo-nextest --version 0.9.57 --locked",
-            },
-        )?;
         setup_flags(build)
     }
 }
